src/Model/Behavior/Strategy/TinyStrategy.php

```diff
@@ -3,27 +3,19 @@
 
 namespace Muffin\Obfuscate\Model\Behavior\Strategy;
 
-use ZackKitzmiller\Tiny;
-
 /**
  * Class TinyStrategy
+ *
+ * @deprecated Use Muffin\Obfuscate\Model\Behavior\Strategy\Base62Strategy instead
  */
 class TinyStrategy implements StrategyInterface
```

Comment:
I am wondering if we should just drop this `TinyStragtegy`. Existing users will have to change the underlying lib anyway, so they can change the strategy name too.

Reply:
You're right, we may drop the TinyStrategy completely